### PR TITLE
Additional overrides

### PR DESCRIPTION
**Pull request by scaronni**

### CODE REVIEW

**Comment by scaronni** [comment]
Builds fine. Please proceed with the merge request if you think it's right.
Thanks.

**Comment by Blystad** [comment]
Can you do a rebase and delete:
```
Wallet local override
Revert "Wallet local override"
```

(`git rebase -i` and then in the editor delete the two commit lines, then force push).

Can you add the new packages to `opengapps-packages.mk` for the correct Package level? See [OpenGapps Package comparison](https://github.com/opengapps/opengapps/wiki/Package-Comparison)

File: modules/CalculatorGoogle/Android.mk [hunk 1]
```diff
@@ -0,0 +1,11 @@
+LOCAL_PATH := .
+include $(CLEAR_VARS)
+LOCAL_MODULE := CalculatorGoogle
+LOCAL_PACKAGE_NAME := com.google.android.calculator
+LOCAL_MODULE_PATH := $(PRODUCT_OUT)/system/app
```
Comment (Blystad) [inline modules/CalculatorGoogle/Android.mk]: Please remove this line. Ref #20

File: modules/TagGoogle/Android.mk [hunk 2]
```diff
@@ -0,0 +1,11 @@
+LOCAL_PATH := .
+include $(CLEAR_VARS)
+LOCAL_MODULE := TagGoogle
+LOCAL_PACKAGE_NAME := com.google.android.tag
+LOCAL_MODULE_PATH := $(PRODUCT_OUT)/system/app
```
Comment (Blystad) [inline modules/TagGoogle/Android.mk]: Please remove this line. Ref #20

**Comment by scaronni** [comment]
Sorry for the late reply, I was on a leave.

**Comment by scaronni** [comment]
Should TagGoogle have `LOCAL_PRIVILEGED_MODULE`? What is the logic here to know if it's required or not?

Thanks.

**Comment by Blystad** [comment]
Privileged module just means it should be placed in /system/priv-app instead of /system/app

**Comment by scaronni** [comment]
Anything else I should do on my side?

**Comment by Blystad** [comment]
Merged. Thanks for your contributions.

**Pull request merged**